src/components/UncontrolledTabs.js

```diff
@@ -157,7 +158,7 @@ export default class UncontrolledTabs extends Component {
         }
 
         result = cloneElement(child, {
-          children: React.Children.map(child.props.children, tab => {
```

Comment:
When we use deepMap, we could actually remove the nested map here, as we would visit the Tab anyway in the outer map?